### ISSUE TITLE
1.3.2 服务长时间关闭后，启动后无法注册

### ISSUE THREAD

**Issue by lin-mt** [alibaba/nacos#4054]
**Describe the bug**
在服务注册到 nacos 之后，手动关停服务，等待一定时间后，重启服务，服务无法注册到 nacos
After the service is registered to nacos, manually shut down the service, wait a certain period of time, and then restart the service, the service cannot be registered to nacos

**Expected behavior**
服务重启后可以正常注册到 nacos
After the service restarts, you can register to nacos normally

**Acutally behavior**
控制台一直循环打印注册请求，返回 "ok"，并且无法注册到 nacos
The console keeps printing the registration request in a loop, returns "ok", and cannot register to nacos

**How to Reproduce**
1. 以 standalone 模式启动 nacos 1.3.2
Start nacos 1.3.2 in standalone mode
2. 配置服务 
config service
```
spring:
  cloud:
    nacos:
      discovery:
        server-addr: 127.0.0.1:8848
```
3. 启动服务，服务成功注册
Start the service, the service is successfully registered
4. 手动关闭服务
Turn off the service manually
5. 等待一定时间（10分钟左右，没有具体算，如果无法复现，时间可以等长一点）
Wait for a certain time (about 10 minutes, no specific calculation, if it cannot be reproduced, the time can be longer)
6. 重新启动服务，服务一直循环打印注册参数信息
Restart the service, the service has been printing the registration parameter information in a loop
![image](https://user-images.githubusercontent.com/58337359/97097900-93d27d80-16b1-11eb-8a53-87d9cde702c1.png)
7. 服务重启后，nacos 服务状态
Service after restarting, nacos service status
![image](https://user-images.githubusercontent.com/58337359/97097912-a5b42080-16b1-11eb-84a1-a02766393a12.png)
8. com.alibaba.nacos.client.naming.net.NamingProxy.registerService 请求结果（request result）
![image](https://user-images.githubusercontent.com/58337359/97097925-ddbb6380-16b1-11eb-9ba5-a85c7f946899.png)

**Desktop (please complete the following information):**
 - OS: Mac OS
 - Version 
    nacos-1.3.2
 - Module client
 - SDK 'springCloudAlibabaVersion', '2.2.3.RELEASE'

**Additional context**
Non

**Comment by KomachiSion** [comment]
Please use 1.4.0-BETA to retry. It might a bug fixed in 1.4.0.

**Comment by KomachiSion** [comment]
Any process？ If no more response, we will think the problem has been fixed in 1.4.0 and close issue.

**Comment by lin-mt** [comment]
> Any process？ If no more response, we will think the problem has been fixed in 1.4.0 and close issue.

嗯，目前没有什么问题
